backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/CsvProcessingException.java

```diff
@@ -1,12 +1,26 @@
 package gov.cdc.usds.simplereport.api.model.errors;
 
+import lombok.Getter;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
 /** An error thrown when CSV uploads fail for reasons other than validation */
 @ResponseStatus(HttpStatus.BAD_REQUEST)
 public class CsvProcessingException extends RuntimeException {
+
+  @Getter private final int lineNumber;
+  @Getter private final int columnNumber;
+
+  public CsvProcessingException(String message, int lineNumber, int columnNumber) {
+    super(message);
+
+    this.lineNumber = lineNumber;
+    this.columnNumber = columnNumber;
+  }
+
   public CsvProcessingException(String message) {
     super(message);
+    this.lineNumber = 0;
+    this.columnNumber = 0;
```

Comment:
I think it might make sense to make these capital `I` Integers so we can just make these null.  They have to be initialized because intellij wanted the fields to be final (which makes sense you don't want dynamic state in an exception attribute).  Setting them to 0 was intended to indicate "not set" but obviously that's not super clear